src/main/java/seedu/address/logic/commands/TagCommandGroup.java

```diff
@@ -8,7 +8,8 @@
 public abstract class TagCommandGroup extends Command {
     public static final String COMMAND_GROUP = "tag";
     public static final String COMMAND_GROUP_ALIAS = "t";
-    public static final String[] BANNED_TAG_NAMES = new String[]{"create", "c", "remove", "r", "delete", "d"};
+    public static final String[] BANNED_TAG_NAMES = new String[] {
+        "create", "c", "remove", "r", "delete", "d", "list", "l"};
```

Comment:
Missed this in the previous PR. Would using the static strings from the class help with code maintainability?
```suggestion
    public static final String[] BANNED_TAG_NAMES = new String[] {
        CreateTagCommand.COMMAND_SPECIFIER, CreateTagCommand.COMMAND_SPECIFIER_ALIAS, 
        RemoveTagCommand.COMMAND_SPECIFIER, RemoveTagCommand.COMMAND_SPECIFIER_ALIAS,
        DeleteTagCOmmand.COMMAND_SPECIFIER, DeleteTagCommand.COMMAND_SPECIFIER_ALIAS,
        TagListCommand.COMMAND_SPECIFIER, TagListCommand.COMMAND_SPECIFIER_ALIAS};
```

Reply:
Agree